index.js

```diff
@@ -38,6 +38,19 @@ var argv = require('yargs')
     default: false,
     global: true
   })
+  .option('pre-release', {
+    alias: 'p',
+    describe: 'Should the release be a pre-release?',
+    type: 'boolean',
+    default: false,
+    global: true
+  })
+  .option('pre-id', {
```

Comment:
I'm not completely sure about the relationship between git tag and npm publish tag. It doesn't look the same to me. npm publish by default uses "latest" tag.